src/CfgMgr32/CfgMgr32+CM_NOTIFY_FILTER_ACTION.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="CM_NOTIFY_FILTER_FLAG"/> nested enum.
+    /// </content>
+    public static partial class CfgMgr32
+    {
+        /// <summary>
+        /// This enumeration identifies Plug and Play device event types.
+        /// </summary>
+        /// <seealso href="https://docs.microsoft.com/en-us/windows/win32/api/cfgmgr32/ne-cfgmgr32-cm_notify_action"/>
+        public enum CM_NOTIFY_ACTION
+        {
+            /* Filter type: CM_NOTIFY_FILTER_TYPE_DEVICEINTERFACE */
+
+            /// <summary>
+            /// For this value, set <see cref="CM_NOTIFY_FILTER.FilterType"/>
+            /// to CM_NOTIFY_FILTER_TYPE_DEVICEINTERFACE. This action indicates that a device interface that meets your filter criteria has been enabled.
+            /// </summary>
+            CM_NOTIFY_ACTION_DEVICEINTERFACEARRIVAL = 0,
```

Comment:
Would the `CM_NOTIFY_ACTION_` prefix normally be removed?

Reply:
No.  See https://github.com/dotnet/pinvoke/blob/master/CONTRIBUTING.md#naming

Reply:
Thanks.